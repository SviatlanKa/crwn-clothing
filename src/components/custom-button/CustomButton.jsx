import React from 'react';
import { CustomButtonContainer } from './CustomButtonStyled';
import './CustomButton.scss';

const CustomButton = ({ children, ...props }) => (
    <CustomButtonContainer { ...props }>
        {children}
    </CustomButtonContainer>
);

export default CustomButton;