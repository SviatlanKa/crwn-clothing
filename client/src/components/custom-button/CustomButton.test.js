import React from 'react';
import { shallow } from 'enzyme';
import CustomButton from './CustomButton';

it('expect render CartDropdown component', () => {
    const wrapper = shallow(<CustomButton inverted children isGoogleSignIn/>);
    expect(wrapper).toMatchSnapshot();
});