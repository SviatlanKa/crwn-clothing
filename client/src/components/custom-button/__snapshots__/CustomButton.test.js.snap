// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render CartDropdown component 1`] = `
<button
  className="inverted google-sign-in custom-button"
>
  <Component />
</button>
`;
